.eslintrc.cjs

```diff
@@ -69,5 +69,11 @@ module.exports = {
 
 		// These on-by-default rules don't work well for this repo and we like them off.
 		"no-inner-declarations": "off",
+
+		// Formatting rule - does not interfere with Prettier
+		"padding-line-between-statements": [
```

Comment:
Ah, you'll actually want `@typescript-eslint/padding-line-between-statements`. https://typescript-eslint.io/linting/troubleshooting/#i-am-using-a-rule-from-eslint-core-and-it-doesnt-work-correctly-with-typescript-code

```suggestion
		"@typescript-eslint/padding-line-between-statements": [
```